pages/agent/v3/elastic_ci_aws/managing_elastic_ci_stack.md

```diff
@@ -87,14 +87,14 @@ or a specific release available from the [releases page](https://github.com/buil
 The latest stable release can be deployed to any of our supported AWS Regions.
 
 The most recent build of the CloudFormation stack is published to
-`https://s3.amazonaws.com/buildkite-aws-stack/master/aws-stack.yml`, along with
+`https://s3.amazonaws.com/buildkite-aws-stack/main/aws-stack.yml`, along with
 a version for each commit at
-`https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
+`https://s3.amazonaws.com/buildkite-aws-stack/main/${COMMIT}.aws-stack.yml`.
+Note: for commits that are ancestors of [419f271b54802c4c8301730bc35b34ed379074c4](https://github.com/buildkite/elastic-ci-stack-for-aws/commit/419f271b54802c4c8301730bc35b34ed379074c4), the build was published to `https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
```

Comment:
Like you mentioned, this formatting feels a bit awkward. I think it's okay to leave as-is, but I'd probably make it a proper note:

```suggestion

>📘 Ancestors of commit 419f271
> For commits that are ancestors of [419f271](https://github.com/buildkite/elastic-ci-stack-for-aws/commit/419f271b54802c4c8301730bc35b34ed379074c4), the build was published to `https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml`.
```

I also used the shortened commit hash to make it easier to read.

An alternative you could consider that might not draw quite as much attention would be an expand:

```suggestion

<details>
  <summary> 📝 Ancestors of commit 419f271</summary>
  <p>For commits that are ancestors of <a href="https://github.com/buildkite/elastic-ci-stack-for-aws/commit/419f271b54802c4c8301730bc35b34ed379074c4">419f271</a>, the build was published to:
  <p><pre>https://s3.amazonaws.com/buildkite-aws-stack/master/${COMMIT}.aws-stack.yml</pre></p>
</details>
```

I think I'd go with the standard note format.